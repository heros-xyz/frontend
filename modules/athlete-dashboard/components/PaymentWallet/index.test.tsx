import { render } from "@testing-library/react";
import Payment from ".";

it("renders homepage unchanged", () => {
  const onChangeStep = (step: number) => {};
  const { container } = render(<Payment onSubmit={() => onChangeStep(2)} />);
  expect(container).toMatchSnapshot();
});
