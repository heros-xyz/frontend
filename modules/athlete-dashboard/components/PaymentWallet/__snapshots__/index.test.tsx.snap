// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders homepage unchanged 1`] = `
<div>
  <div
    class="css-1m1yvh9"
  >
    <div
      class="css-soukc5"
    >
      <svg
        class="chakra-icon css-1vmpetx"
        focusable="false"
        viewBox="0 0 32 32"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M26 16L6 16M6 16L15 7M6 16L15 25"
          stroke="#1E16C1"
          stroke-linecap="round"
          stroke-linejoin="round"
          strokeWidth="2"
        />
      </svg>
      <span
        class="chakra-text css-141yne6"
      >
        Payment Information
      </span>
    </div>
    <p
      class="chakra-text css-f4ncpt"
    >
      Payment Method
    </p>
    <div
      class="css-13rbvqk"
    >
      <button
        class="chakra-button css-1ofr8m2"
        type="button"
      >
        Update payment method
      </button>
    </div>
  </div>
</div>
`;
